build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:7.0.0-alpha05'
```

Comment:
Any reason to use the alpha version here? Android Studio still creates new projects with `4.2.1`.